tests/integration/suite_test.go

```diff
@@ -119,6 +124,39 @@ var _ = BeforeSuite(func() {
 		Expect(err).ToNot(HaveOccurred())
 	}()
 
+	nsOpenshiftOperators := &v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "openshift-operators",
+		},
+	}
+	err = k8sClient.Create(context.TODO(), nsOpenshiftOperators, &client.CreateOptions{})
+	Expect(err).NotTo(HaveOccurred())
+
+	odfClientInfoConfigMap := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "odf-client-info",
+			Namespace: "openshift-operators",
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					APIVersion: viewv1beta1.GroupVersion.String(),
+					Kind:       "ManagedClusterView",
+					Name:       "mcv-1",
+					UID:        "mcv-uid",
+				},
+			},
+		},
+		Data: map[string]string{
+			"mc-1_test-storagecluster1":                    "{\"providerInfo\":{\"version\":\"4.19.0\"}}",
+			"mc-2_test-storagecluster2":                    "{\"providerInfo\":{\"version\":\"4.19.0\"}}",
+			"test-provider-cluster1_test-storagecluster-1": "{\"providerInfo\":{\"version\":\"4.19.0\"}}",
+			"test-provider-cluster2_test-storagecluster-2": "{\"providerInfo\":{\"version\":\"4.19.0\"}}",
+			"cluster1_test-storagecluster":                 "{\"providerInfo\":{\"version\":\"4.19.0\"}}",
+			"cluster2_test-storagecluster":                 "{\"providerInfo\":{\"version\":\"4.19.0\"}}",
```

Comment:
Would be preferable to store them as variables rather than hardcoding the values. In case of changes to tests in future

Reply:
In my recent experiences, in case of changes to tests, it's much easier to change it here.
This could technically be removed if we we're creating the config dynamically like in the real use cases. But, we are not. So this is a hack until we have a better test strategy in place.